### ISSUE TITLE
Liquibase's ClassLoaderResourceAccessor should be limited to local classpath only

### ISSUE THREAD

**Issue by githubapreinders** [ibissource/iaf#2447]
**Liquibase initialization fails**

**Ap Reinders**

**To Reproduce**
Application starts up normally but without proper initialisation by Liquibase. In the systemOut.log you can see the application trying to find the proper initialisation files, walking along the whole classpath and giving errors in the logging.

**Environment**
FF! 7.6.1: ibis4adn 20211112_1330-7c5ba44a-master-20
Running on srdzzapp0455 using IBM WebSphere Application Server/9.0.5.7
Java Version: Java(TM) SE Runtime Environment (8.0.6.26 - pxa6480sr6fp26-20210216_01(SR6 FP26))
Heap size: 327M, total JVM memory: 512M
Free disk space: 2GB, total disk space: 2GB
Up since: 2021-11-12 14:40:03 (2d)

**Logging**
[SystemOut_ADN.log](https://github.com/ibissource/iaf/files/7538039/SystemOut_ADN.log)

**Comment by nielsm5** [comment]
Liquibase's ClassLoaderResourceAccessor should be limited to local classpath only
```
java.security.AccessControlException: Access denied ("java.io.FilePermission" "/opt/WAS/properties/" "read")
	at java.security.AccessController.throwACE(AccessController.java:176)
	at java.security.AccessController.checkPermissionHelper(AccessController.java:238)
	at java.security.AccessController.checkPermission(AccessController.java:385)
	at java.lang.SecurityManager.checkPermission(SecurityManager.java:562)
	at com.ibm.ws.security.core.SecurityManager.checkPermission(SecurityManager.java:208)
	at sun.misc.URLClassPath.check(URLClassPath.java:743)
	at sun.misc.URLClassPath.checkURL(URLClassPath.java:723)
	at java.net.URLClassLoader$2.next(URLClassLoader.java:961)
	at java.net.URLClassLoader$2.hasMoreElements(URLClassLoader.java:976)
	at java.lang.ClassLoader$CompoundEnumeration.hasMoreElements(ClassLoader.java:632)
	at com.ibm.ws.classloader.CompoundClassLoader.findParentResources(CompoundClassLoader.java:1254)
	at com.ibm.ws.classloader.CompoundClassLoader.getResources(CompoundClassLoader.java:1198)
	at com.ibm.ws.classloader.CompoundClassLoader.findParentResources(CompoundClassLoader.java:1252)
	at com.ibm.ws.classloader.CompoundClassLoader.getResources(CompoundClassLoader.java:1208)
	at com.ibm.ws.classloader.CompoundClassLoader.getResources(CompoundClassLoader.java:1180)
	at liquibase.resource.AbstractResourceAccessor.init(AbstractResourceAccessor.java:39)
	at liquibase.resource.ClassLoaderResourceAccessor.<init>(ClassLoaderResourceAccessor.java:24)
	at liquibase.servicelocator.ServiceLocator.<init>(ServiceLocator.java:61)
	at liquibase.servicelocator.ServiceLocator.<clinit>(ServiceLocator.java:44)
	at liquibase.logging.LogFactory.getLog(LogFactory.java:46)
	at liquibase.logging.LogFactory.getLog(LogFactory.java:72)
	at liquibase.database.DatabaseFactory.<init>(DatabaseFactory.java:27)
	at liquibase.database.DatabaseFactory.getInstance(DatabaseFactory.java:48)
	at liquibase.Liquibase.<init>(Liquibase.java:85)
```

**Comment by nielsm5** [comment]
I don't think there is much we can do. Liquibase scans the JVM directories for files/configurations. Since WebSphere 9 you will see these warnings, unless you add them to the was.policy file.

**Comment by nielsm5** [comment]
The stacktraces provided are indeed caused by liquibase. They however should not impact the functionality of liquibase.
We will not fix the AccessControlExceptions in the 7.6 release branch.
